InfluxDBv2_Covid19_SouthAmerica/covid.yml

```diff
@@ -0,0 +1,598 @@
+                from(bucket: "covid")
+                  |> range(start: v.timeRangeStart, stop: v.timeRangeStop)
+                  |> filter(fn: (r) => r["_measurement"] == "exec_paraguay")
+                  |> filter(fn: (r) => r["_field"] == "location_latest_confirmed")
+        width: 2
+        xPos: 8
+      - colors:
+          - hex: '#ffffff'
+            name: white
+            type: text
+        decimalPlaces: 2
+        height: 1
+        kind: Single_Stat
+        name: Nuevos casos 24 horas
+        queries:
+          - query: |-
+                from(bucket: "covid")
+                  |> range(start: -24h)
+                  |> filter(fn: (r) => r["_measurement"] == "exec_paraguay")
+                  |> filter(fn: (r) => r["_field"] == "location_latest_confirmed")
+                  |> increase()
+                  |> yield(name: "increase")
+        width: 2
+        xPos: 8
+        yPos: 1
+      - colors:
+          - hex: '#00C9FF'
+            name: laser
+            type: text
+        decimalPlaces: 2
+        height: 1
+        kind: Single_Stat
+        name: Uruguay
+        queries:
+          - query: |-
+                from(bucket: "covid")
+                  |> range(start: v.timeRangeStart, stop: v.timeRangeStop)
+                  |> filter(fn: (r) => r["_measurement"] == "exec_uruguay")
+                  |> filter(fn: (r) => r["_field"] == "location_latest_confirmed")
+        width: 2
+        xPos: 10
+      - colors:
+          - hex: '#ffffff'
+            name: white
+            type: text
+        decimalPlaces: 2
+        height: 1
+        kind: Single_Stat
+        name: Nuevos casos 24 horas
+        queries:
+          - query: |-
+                from(bucket: "covid")
+                  |> range(start: -24h)
+                  |> filter(fn: (r) => r["_measurement"] == "exec_uruguay")
+                  |> filter(fn: (r) => r["_field"] == "location_latest_confirmed")
+                  |> increase()
+                  |> yield(name: "increase")
+        width: 2
+        xPos: 10
+        yPos: 1
+    name: COVID19 América del Sur y Resto del mundo
+---
+
+apiVersion: influxdata.com/v2alpha1
+kind: Telegraf
+metadata:
+    name: covid19-southamerica
+spec:
+    associations:
+      - kind: Label
+        name: vigorous-pike-607001
+    config: |
+         [[outputs.influxdb_v2]]
+         ## The URLs of the InfluxDB cluster nodes.
+         ##
+           ## Multiple URLs can be specified for a single cluster, only ONE of the
+           ## urls will be written to each interval.
+           ## urls exp: http://127.0.0.1:9999
+           urls = ["$INFLUX_HOST"]
+         
+           ## Token for authentication.
+           token = "$INFLUX_TOKEN"
+         
+           ## Organization is the name of the organization you wish to write to; must exist.
+           organization = "$INFLUX_ORG"
+         
+           ## Destination bucket to write into.
+           bucket = "$INFLUX_BUCKET"
```

Comment:
Since the template provides a bucket named "covid" now, it can be hard-coded here. Otherwise the README should instruct the user to explicitly set INFLUX_BUCKET="covid" in their environment for Telegraf.

Reply:
Thanks for pointing that. I just add that instruction to the readme to remain untouched the template.